roles/Dev/files/profile

```diff
@@ -38,10 +38,34 @@ function theme {
 	NAME=$1; if [ -z "$NAME" ]; then NAME="Peppermint"; fi
 	osascript -e "tell application \"Terminal\" to set current settings of front window to settings set \"$NAME\""
 }
+
 # Switches theme when SSHing
-# TODO: have this function parse .ssh/config and look for Theme declaration to switch to per ssh config or use Steve's thing
+# Parse .ssh/config to set theme using #Theme declaration (#Theme since .ssh/config only accepts valid config options):
+# Host sample.local.dev
+#   #Theme Red Sands
```

Comment:
I don't like that we have to use `#Theme` here but if you use an invalid configuration option your entire `.ssh/config` file breaks and any ssh operation results in this message:

```
/Users/bogdan/.ssh/config: line 13: Bad configuration option: theme
```